src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -163,8 +163,11 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             src="{{workItem.relationalData.creator.attributes.imageURL + '&s=20'}}"
             id="WI_details_reporter_img" />          
           <div class="pull-left marginR10">                          
-              <span id="WI_details_reporter_user"> 
-                {{workItem.relationalData?.creator?.attributes.fullName || 'No creator found'}} 
```

Comment:
I think the proper solution here is to allow **API to provide the username string as well**, and change line 167 into:

```js
{{ workItem.relationalData.creator.attributes.fullName || workItem.relationalData.creator.attributes.userName }}
```

I've removed the check for the existence of the nested objects, guessing that the returned successful API response for a WI will always have a creator - if I'm wrong to assume that, please add back the safety-guards (i.e. `workItem.relationalData?.creator?.attributes.keyName`)